packages/platforms/delta/armel/arm-delta-ag6248c/arm-delta-ag6248c-poe/modules/builds/arm-delta-ag6248c-poe-cpld-mux-1.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * An I2C multiplexer dirver for delta as5812 CPLD
+ *
+ * Copyright (C) 2015 Delta Technology Corporation.
+ * Brandon Chuang <brandon_chuang@delta.com.tw>
```

Comment:
This was clearly an s/[Aa]ccton/[Dd]elta/g. Brandon Chuang works for Accton.

Reply:
the comments is updated, please check it in the commit b1eeddcc3186b27c5066977cfd3687c061bca952